### PR TITLE
Feature/initial commit

### PR DESCRIPTION
**Pull request by lats1**
https://jira.itkdev.dk/browse/ODAA-61

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__.'/{src,tests}')
+;
+
+$header =<<<'HEADER'
+This file is part of itk-dev/odaa.
```
Comment (rimi-itk) [inline .php_cs.dist]: Shouldn't this be `itk-dev/datatidy`?

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2011-2019 ITK Development.
```
Comment (rimi-itk) [inline LICENSE.md]: This should probably be just `2019`.

File: README.md [hunk 3]
```diff
@@ -1 +1,113 @@
...
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Installing
+
+```bash
+$ docker-compose pull
+$ docker-compose up -d
+
+$ docker-compose exec phpfpm bin/install
+$ docker-compose exec phpfpm bin/console doctrine:database:create
```
Comment (rimi-itk) [inline README.md]: The database already exists in the `docker-compose` setup.

File: README.md [hunk 4]
```diff
@@ -1 +1,113 @@
...
+$ docker-compose exec phpfpm bin/console doctrine:migrations:migrate
+
+$ docker run -v ${PWD}:/app itkdev/yarn:latest install
+$ docker run -v ${PWD}:/app itkdev/yarn:latest encore dev
+```
```
Comment (rimi-itk) [inline README.md]: I prefer not having `$` in shell code blocks. Leaving it out makes it easier to copy whole lines and run them in a shell.

File: README.md [hunk 5]
```diff
@@ -1 +1,113 @@
...
+
+### Installing
+
+```bash
+$ docker-compose pull
+$ docker-compose up -d
+
+$ docker-compose exec phpfpm bin/install
+$ docker-compose exec phpfpm bin/console doctrine:database:create
+$ docker-compose exec phpfpm bin/console doctrine:migrations:migrate
```
Comment (rimi-itk) [inline README.md]: Maybe add `--no-interaction`?

File: README.md [hunk 6]
```diff
@@ -1 +1,113 @@
-# ODAA
+# Datatidy
```
Comment (rimi-itk) [inline README.md:1]: A really well done README full of useful information. Good job!

File: README.md [hunk 7]
```diff
@@ -1 +1,113 @@
...
+- [Docker](https://docs.docker.com/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Installing
+
+```bash
+$ docker-compose pull
+$ docker-compose up -d
+
+$ docker-compose exec phpfpm bin/install
```
Comment (rimi-itk) [inline README.md]: `bin/install`? Does this script exist?

File: config/packages/doctrine.yaml [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+    dbal:
+        # configure these for your database server
+        driver: 'pdo_mysql'
+        server_version: '5.7'
+        charset: utf8mb4
+        default_table_options:
+            charset: utf8mb4
+            collate: utf8mb4_unicode_ci
+
+        url: '%env(resolve:DATABASE_URL)%'
```
Comment (rimi-itk) [inline config/packages/doctrine.yaml:11]: When/if we start using a database for storing data sets and transformed data, we probably want to set `schema_filter` here to exclude custom data source/transform tables (cf. https://symfony.com/doc/master/bundles/DoctrineMigrationsBundle/index.html#manual-tables). Alternatively, we could use another database connection for the actual data wrangling.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (lats1) [reply README.md]: Heh well not yet - now using good old composer with 723cfd2

File: README.md (hunk 4 above)
Comment (lats1) [reply README.md]: Good point. Fixed in 88e6928

File: .php_cs.dist (hunk 1 above)
Comment (lats1) [reply .php_cs.dist]: It surely should be. Fixed in 7fa9cf7

File: LICENSE.md (hunk 2 above)
Comment (lats1) [reply LICENSE.md]: Fixed in 5b3143c

File: README.md (hunk 3 above)
Comment (lats1) [reply README.md]: Fixed in 3481089

File: README.md (hunk 5 above)
Comment (lats1) [reply README.md]: Fixed in 3481089

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**